routes/drinks.js

```diff
@@ -0,0 +1,276 @@
+import express from 'express';
+import { createCloudinaryFolder, setupImageFolders, uploadImageToFolder } from '../db_config/cloudinary_config.js';
+import pool from '../db_config/db_connection.js'
+import { config } from 'dotenv';
+config();
+
+import swaggerUi from 'swagger-ui-express';
+import YAML from 'yamljs';
+
+const router = express.Router();
+
+const swaggerDocument = YAML.load('./documentary/swagger-specs.yaml');
+
+// mounting the Swagger UI middleware:
+router.use('/api-docs', swaggerUi.serve);
+router.get('/api-docs', swaggerUi.setup(swaggerDocument));
+
+
+async function uploadDrinkImage(imagePath) {
+  await setupImageFolders(); // Ensure the 'drinks-images' folder exists
+  const uploadResult = await uploadImageToFolder(imagePath, 'drinks-images');
+  console.log('Uploaded image:', uploadResult.secure_url);
+}
+
+// Call the function to upload an image
+//uploadDrinkImage('./images/purple.png');
+
+
+// Get all recipes
+router.get('/', async (req, res) => {
+  try {
+    const query = 'SELECT * FROM Drinks';
+    const { rows } = await pool.query(query);
+    res.json(rows);
+  } catch (error) {
+    // res.status(500).json({ error });
+    //throw err;
+  }
+});
+
+
+// Get a recipe by ID
+router.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const recipe = await getDrinkById(id);
+    if (!recipe) {
+      return res.status(404).json({ error: 'drink not found' });
+    }
+    res.json(recipe);
+  } catch (err) {
+    console.error('Error retrieving drink:', err);
+    res.status(500).json({ error: 'message' });
```

Comment:
`next(error)` as such the error handling middleware of express handles the error.